action.yml

```diff
@@ -19,8 +19,11 @@ inputs:
     description: Should the setup.ini file signature be checked?
     required: false
     default: true
+  pubkeys:
+    description: Absolute path of extra public key files (RFC4880 format)
+    required: false
   site:
-    description: Download site URL
+    description: Download site URLs separated by space
```

Comment:
Perhaps this should say "whitespace", as I think that's what the split is doing, and newline separating them might look nicer